runtime/src/integration/content_directory.rs

```diff
@@ -0,0 +1,47 @@
+use crate::{AccountId, MemberId, Runtime};
+
+// The content directory working group instance alias.
```

Comment:
It is already taken. Take instance3 instead. Also to prevent future errors - please move this line to the lib.rs closer to the Instance2 declaration.

Reply:
Ok. But i saw, both service-discovery and storage modules are using `Instance2` Is that correct case?

Reply:
Done here https://github.com/Joystream/joystream/pull/1273/commits/b9c741ed8b0adecd6ee1b13e67fba8604574e787

Reply:
Yes. Because they use the same working group.